firmware/boron-ins-fsm/src/stateMachine.h

```diff
@@ -12,13 +12,15 @@
 // which is also unlikely to be part of a door ID or a threshold/timer const
 #define INITIALIZE_STATE_MACHINE_CONSTS_FLAG           0x8888
 #define INITIALIZE_STATE3_MAX_LONG_STILLNESS_TIME_FLAG 0x8888
+#define INITIALIZE_STATE0_WINDOW_FLAG                  0x8888
 
 // initial (default) values for state machine, can be changed via console function
 // or by writing something other than 0x8888 to the above flag in flash
-#define INS_THRESHOLD             60
-#define STATE1_MAX_TIME           15000    // ms = 15s
-#define STATE2_MAX_DURATION       1200000  // ms = 20 min
-#define STATE3_MAX_STILLNESS_TIME 120000   // ms = 2 minutes
+#define INS_THRESHOLD                   60
+#define STATE0_WINDOW_TIME              3600000  // ms = 60 min
```

Comment:
From what we talked about earlier, I was expecting a higher default value. Why did you end up choosing 60 minutes?

Reply:
I don't think there is much risk in immediately implementing a 1 hour timeout on all sensors to alleviate all middle of the night alerts (acting basically as the "hours of operation" fix), without really changing how production sensors are operating currently. Do you think this would be too risky?

Reply:
No, I believe that was Duration alerts repeating. For Edmonton, I plan to decrease the state0 timer even further to maybe 10 minutes and reduce the initial timer to 5s.

People can definitely be occupying the bathroom for more than 1 hour, but if the sensor hasn't picked them up by an hour, I don't think it will in the next hour either. I would prefer to start with 1 hour.